packages/react/src/components/select/select.stories.tsx

```diff
@@ -0,0 +1,134 @@
+import {
+  Field,
+  FieldLabel,
+  HelperText,
+  Select,
+  SelectProps,
+  Validation,
+} from '@onfido/castor-react';
+import React from 'react';
+import { Meta, omit, reactMatrix, Story } from '../../../../../docs';
+
+const borderless = [true, false] as const;
+const disabled = [true, false] as const;
+const invalid = [true, false] as const;
+
+export default {
+  title: 'React/Select',
+  component: Select,
+  argTypes: {
+    children: {
+      description: 'List of options using `<option>`.',
+      control: false,
+    },
+    borderless: {
+      table: { type: { summary: 'boolean' } },
+    },
+    disabled: {
+      table: { type: { summary: 'boolean' } },
+    },
+    invalid: {
+      table: { type: { summary: 'boolean' } },
+    },
+    placeholder: {
+      table: { type: { summary: 'string' } },
+    },
```

Comment:
🤔  I'm surprised these can't be picked up by `storybook-react-props` or whatever that lib is called.

Reply:
It does not pick the "type" for the table.

I believe this is because we use a `withRef`.

Reply:
I wonder if `forwardRef` can be picked 🤔 
Something to check later